lib/screenplay_web/plugs/metadata.ex

```diff
@@ -33,13 +33,19 @@ defmodule ScreenplayWeb.Plugs.Metadata do
     |> assign(:screens_url, Application.get_env(:screenplay, :screens_url))
     |> assign(:signs_ui_url, Application.get_env(:screenplay, :signs_ui_url))
     |> assign(:is_admin, admin?(conn))
```

Comment:
If you're open to it, mind changing the references to `is_admin` into `is_emergency_admin` for clarity?

Reply:
Sure thing!